platform/viewer/cypress/integration/measurement-tracking/OHIFStudyViewer.spec.js

```diff
@@ -278,37 +278,37 @@ describe('OHIF Study Viewer Page', function() {
 
     //Right click on viewport
     cy.get('@viewport')
-      .trigger('mousedown', 'top', { which: 3 })
-      .trigger('mousemove', 'center', { which: 3 })
+      .trigger('mousedown', 'top', { buttons: 2 })
+      .trigger('mousemove', 'center', { buttons: 2 })
       .trigger('mouseup');
 
-    const expectedText = 'Zoom:3.98x';
+    const expectedText = 'Zoom:6.65x';
     cy.get('@viewportInfoTopLeft').should('contains.text', expectedText);
   });
 
-  it('performs middle click to pan', function() {
+  /*it('performs middle click to pan', function() {
```

Comment:
is it commented out because it doesn't work?

Reply:
because it was checking the `translation` on the `viewport` which doesn't exist anymore